pkgs/top-level/all-packages.nix

```
@@ -28097,6 +28097,10 @@ with pkgs;
 
   cascadia-code = callPackage ../data/fonts/cascadia-code { };
 
+  catboost = callPackage ../development/libraries/catboost {
+    stdenv = clang12Stdenv;
```

clang12Stdenv is an alias, I recommend to turn them off if you are contributing a lot.

```suggestion
    inherit (llvmPackages_12) stdenv;
```